content/en/docs/Responses/AlbumID3.md

```diff
@@ -11,19 +11,59 @@ description: >
 {{< tab header="**Example**:" disabled=true />}}
 {{< tab header="OpenSubsonic" lang="json">}}
 {
-  "id": "ad0f112b6dcf83de5e9cae85d07f0d35",
-  "name": "8-bit lagerfeuer",
-  "artist": "pornophonique",
-  "year": 2007,
-  "coverArt": "al-ad0f112b6dcf83de5e9cae85d07f0d35_640a93a8",
-  "starred": "2023-03-22T01:51:06Z",
-  "duration": 1954,
-  "playCount": 97,
-  "created": "2023-03-10T02:19:35.784818075Z",
-  "artistId": "91c3901ac465b9efc439e4be4270c2b6",
-  "songCount": 8,
-  "played": "2023-03-28T00:45:13Z",
-  "userRating": 4
+    "id": "ad0f112b6dcf83de5e9cae85d07f0d35",
+    "name": "8-bit lagerfeuer",
+    "artist": "pornophonique",
+    "year": 2007,
+    "coverArt": "al-ad0f112b6dcf83de5e9cae85d07f0d35_640a93a8",
+    "starred": "2023-03-22T01:51:06Z",
+    "duration": 1954,
+    "playCount": 97,
+    "genre": "Hip-Hop",
+    "created": "2023-03-10T02:19:35.784818075Z",
+    "artistId": "91c3901ac465b9efc439e4be4270c2b6",
+    "songCount": 8,
+    "played": "2023-03-28T00:45:13Z",
+    "userRating": 4,
+    "recordLabel": "Sony",
```

Comment:
recordLabel should probably be an array too (for XML It'll just have to appear multiple times I guess) since albums can have multiple labels, especially for special editions

Reply:
At the tag level no file format support a multi values field.

Reply:
At least per [ID3v2.4 spec](https://github.com/id3/ID3v2.4/blob/516075e38ff648a6390e48aff490abed987d3199/id3v2.4.0-frames.txt#L253), there is formal definition of multi-valued tags (separated using `NULL` byte)

Reply:
> All text information frames supports multiple strings, stored as a null
   separated list, where null is reperesented by the termination code
   for the charater encoding. All text frame identifiers begin with "T".

> TPUB
The 'Publisher' frame simply contains the name of the label or publisher.

At least from my reading there, this would mean that `TPUB` is also supported (I don't see any exceptions from the second definiton), as it is a text file. Mutagen and Taglib both handle this properly as well, although FFMpeg does not, so ymmv.

Reply:
It also varies a lot on the tagger, TRCK, TIT2, TLEN and many other in theory also can be multi valued but no one would know what to do if there was multiple values. Anyway seems at least MP3tag support multi value for TPUB so let's change to array.